src/assets/styles/common.css

```diff
@@ -1760,3 +1760,62 @@ li.dropdown.disabled a {
 	margin-top: 40px;
 }
 
+@media (min-width: 769px){
+  .delegate-monitor-top > div > div {
+    margin: 0 15px;
+  }
+
+  .nav-tabs > li:first-child > a {
+    margin-left: 32px;
+  }
+}
+
+.nav-tabs > li > a {
+  height: 45px;
+  margin-bottom: -1px;
+}
+
+.exchanges .nav-tabs > li.active > a  {
+  height: auto;
+}
+
+.market-watcher .nav-tabs > li:first-child > a {
+  margin-left: 0px;
+}
+
+@media (min-width: 769px) and (max-width: 1024px) {
+  .latest-votes-table {
```

Comment:
The latest votes table does not align well when displayed in a 425px width.

Reply:
Modified html and css to make the table for Latest Votes and Newest Delegates consistent.